### PR TITLE
Revert "fix: disable adding prefix 'M_' and 'R_' in exprotModel function"

### PR DESCRIPTION
**Pull request by edkerk**
tldr; release bugfix version 2.5.2 ASAP, keep changes from #353 for minor release 2.6.0.
 
1. As Issue [#353](https://github.com/SysBioChalmers/RAVEN/issues/353) fundamentally changes the output from model I/O, it should be part of minor release 2.6.0 (instead of bug-fix 2.5.2), as [mentioned](https://github.com/SysBioChalmers/RAVEN/pull/356#issuecomment-895504529).
2. PR [#356](https://github.com/SysBioChalmers/RAVEN/issues/356) only addressed `exportModel`, without changing `importModel` and `checkModelStruct`, as should be done to completely solve Issue [#353](https://github.com/SysBioChalmers/RAVEN/issues/353).
3. Meanwhile, PR [#358](https://github.com/SysBioChalmers/RAVEN/issues/358) is on standby (waiting for PR [#357](https://github.com/SysBioChalmers/RAVEN/issues/357) to be reviewed), to release bugfix version 2.5.2. **This release should be made ASAP, as it contains essential fixes. Please review and merge!**
4. As [#353](https://github.com/SysBioChalmers/RAVEN/issues/353) was now already merged to `devel`, _before_ merging [#358](https://github.com/SysBioChalmers/RAVEN/issues/358), the changes from `exportModel` would already be integrated in the bugfix release 2.5.2, contra to point 1 above.
5. To solve this:

- [ ] Revert PR [#356](https://github.com/SysBioChalmers/RAVEN/issues/356) with this PR.
- [ ] Review and merge PR [#357](https://github.com/SysBioChalmers/RAVEN/issues/357).
- [ ] Review and merge PR [#358](https://github.com/SysBioChalmers/RAVEN/issues/358). After this, release 2.5.2 is finalized on `main`, but no need to wait to continue with the next step:
- [ ] Make new PR from [`fix/exprotModel`](https://github.com/SysBioChalmers/RAVEN/tree/fix/exprotModel) branch to `devel`, recreating PR [#356](https://github.com/SysBioChalmers/RAVEN/issues/356) and merge.

Apologies, I should have realized this earlier before approving [#356](https://github.com/SysBioChalmers/RAVEN/issues/356). Mental note: first merge release-related PRs (`devel` --> `main`) before approving new PRs to `devel`.

### CODE REVIEW

**Comment by mihai-sysbio** [comment]
> Mental note: first merge release-related PRs (devel --> main) before approving new PRs to devel.

An alternative would be to use separate branches from `devel` for feature-freezing, allowing parallel PRs to `devel` as usual.

**Review by mihai-sysbio — APPROVED** [review]
The plan makes sense, and is time-sensitive, so let's help move this forward.

**Comment by edkerk** [comment]
> > Mental note: first merge release-related PRs (devel --> main) before approving new PRs to devel.
> 
> An alternative would be to use separate branches from `devel` for feature-freezing, allowing parallel PRs to `devel` as usual.

Very nice suggestion, will read up about this and amend the contributor guidelines accordingly.
